lib/mp4/transmuxer.js

```diff
@@ -145,7 +147,7 @@ AudioSegmentStream = function(track, options) {
     track.baseMediaDecodeTime = trackDecodeInfo.calculateTrackBaseMediaDecodeTime(
       track, options.keepOriginalTimestamps);
 
-    audioFrameUtils.prefixWithSilence(
+    videoClockCyclesOfSilencePrefixed = audioFrameUtils.prefixWithSilence(
```

Comment:
I guess this is related to the audioTsToVideoTs call below?

Reply:
It's the amount of audio filled, but the value is reported in the video clock instead of the audio clock, just because our timing goes by that.

Reply:
```suggestion
    // amount of audio filled but the value is in video clock rather than audio clock
    videoClockCyclesOfSilencePrefixed = audioFrameUtils.prefixWithSilence(
```